docs/tutorials/index.md

```diff
@@ -23,7 +23,7 @@ title: "Quickstart (local)"
   -->
 
 
-This quickstart gets you started with Apache Druid using the [`micro-quickstart`](../operations/single-server.md#micro-quickstart-4-cpu-16gib-ram) configuration, and introduces you to Druid ingestion and query features.
+This quickstart gets you started with Apache Druid using the [`micro`](../operations/single-server.md#micro-4-cpu-16gib-ram) configuration, and introduces you to Druid ingestion and query features.
```

Comment:
Ah, I meant to suggest updating the tutorial to use `bin/start-druid` rather than the micro config. I don't think we need to mention the size of the machine at all here, since it's automatic now. Something like this is good:

```
This quickstart gets you started with Apache Druid and introduces you to Druid ingestion and query features.
```